docker/openjdk-11/Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-FROM openjdk:11.0.7-jre-slim-buster
+FROM adoptopenjdk/openjdk11:x86_64-ubuntu-jre-11.0.11_9
```

Comment:
Have you scanned the image built with this as the base instead?
`adoptopenjdk/openjdk11:jre-11.0.11_9`

That is base is architecture agnostic (and is a synonym for ubuntu-jre-11.0.11-9)

Reply:
Sure, I'll give a try with `adoptopenjdk/openjdk11:jre-11.0.11_9`

Reply:
Fixed

Reply:
Done